db/seeds.rb

```diff
@@ -8,14 +8,82 @@
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
 
-# Create the default user
-user = User.create_or_find_by!(
-  first_name: "Jeremy", last_name: "Caley", email: "jeremy",
-  password: "password", password_confirmation: "password"
-)
+require 'faker'
 
-user.workspaces.create_or_find_by!(name: "Caley", owner_id: user.id)
-puts "Workspace created: #{user.workspaces.first.name}"
+email_providers = ['gmail.com', 'yahoo.com', 'hotmail.com', 'outlook.com', 'aol.com']
 
-user.teams.create_or_find_by!(name: "Caley", workspace_id: user.workspaces.first.id)
-puts "Team created: #{user.teams.first.name}"
+# Create multiple users
+5.times do
```

Comment:
I would prefer to have single user with multiple email_servers at moment which might be the primary use case for most users to check how to toggle between multiple email accounts they own.

It will be helpful to have a static email like `user@caley.to` as seed user, so we can document that in readme for other devs to get started quickly without the need to find the login email to try out the app

Reply:
Agree here for now the primary use case is going to be a single user with multiple email_servers